moncli/types.py

```diff
@@ -227,6 +227,29 @@ def validate_hour(self,value):
             raise ValidationError('Minute values must be between 0-59, not "{}".'.format(value.minute))
 
 
+class LocationType(MondayType):
+
+    native_type = en.cv.Location
+    allow_casts = (dict,)
+    null_value = {}
+
+    def _cast(self, value):
+        try:
+            return en.cv.Location(lat=value['lat'],lng=value['lng'])
+        except KeyError:
+            raise ConversionError('Cannot convert "{}" to Location.'.format(value))
+
+    def _export(self, value):
+        if value.lat and value.lng:
+            return { 'lat': value.lat,'lng': value.lng }
```

Comment:
I forgot this in the documentation, but the address needs to be in here as well.